include/revng/ADT/GenericGraph.h

```diff
@@ -1498,6 +1498,13 @@ struct GraphTraits<llvm::Inverse<T *>> {
                                                typename T::const_child_iterator,
                                                typename T::child_iterator>;
 
+  using EdgeRef = typename T::Edge &;
+  template<typename Ty, typename True, typename False>
+  using if_const = std::conditional_t<std::is_const_v<Ty>, True, False>;
```

Comment:
Use `IfConst` or `if_const_t`

Reply:
I changed this to `if_const_t` [here](https://github.com/revng/revng/blob/d25a140596ffda169b22a0334654fc7bb27b028d/include/revng/ADT/GenericGraph.h#L1504). While I was at it, I also made the instances a couple of lines (not part of this changeset originally) above coherent with this [here](https://github.com/revng/revng/blob/d25a140596ffda169b22a0334654fc7bb27b028d/include/revng/ADT/GenericGraph.h#L1466)